### ISSUE TITLE
[Feature] Put all TODOs onto Problem view

### ISSUE THREAD

**Issue by jgclark** [jgclark/vscode-todo-highlight#1]
Have an option that when running the `List Annotated Highlights` command, add results into the Problem tab, not the Output tab.
Alternatively, have a new setting `isProblem` per highlight, that automatically adds them to the Problem tab.

**Comment by xboxfly15** [comment]
Yes please! I'd also like if there was a setting so that the Problems view was updated when the project is opened and a file saved
